RangeShiftR/src/RScore/Landscape.cpp

```diff
@@ -894,6 +894,13 @@ Cell* Landscape::findCell(int x, int y) {
 	else return 0;
 }
 
+bool Landscape::checkDataCell(int x, int y) {
```

Comment:
actually not sure if I really used this function in the end for the management class?

Reply:
Yes, it is used and thus should be fine